tests/configs/test_configs.py

```diff
@@ -4,8 +4,76 @@
 import google.protobuf.text_format as text_format  # weird import for mypy
 import pytest
 from myrtlespeech import configs
+from myrtlespeech.builders.task_config import build
 from myrtlespeech.protos import task_config_pb2
 
+# Utilities -------------------------------------------------------------------
+
+DATASET_STR = "dataset "
+FAKE_DSET = """{
+fake_speech_to_text {
+    dataset_len: 2;
+    audio_ms {
+        lower: 1;
+        upper: 10;
+        }
+    label_symbols: "abc";
+    label_len {
+        lower: 1;
+        upper: 10;
+        }
+    }
+}"""
+
+
+def replace_dataset_w_fake_dataset(config):
```

Comment:
Could this be done by directly modifying the object returned from `text_format.Merge` rather than modifying the underlying file? 

It seems like this could be more reliable than implementing our own parser?

Reply:
Good point - I've altered